src/fe-text/irssi.c

```diff
@@ -74,11 +75,20 @@ void mainwindow_activity_deinit(void);
 void mainwindows_layout_init(void);
 void mainwindows_layout_deinit(void);
 
+#ifdef HAVE_SECCOMP
```

Comment:
Yes, but it would be contained to the relevant file (`sandbox.c`).

Anyway, this is a question of policy related to irssi, and since I'm not an irssi developer, you should ignore me for now and wait for input from @dequis or @ahf to hear their opinion. :-)